giraffe/src/components/geo/PointMapLayer.tsx

```diff
@@ -0,0 +1,201 @@
+// Libraries
+import React, {useEffect, useState} from 'react'
+import {FunctionComponent, useLayoutEffect} from 'react'
+import L from 'leaflet'
+import {Marker} from 'react-leaflet'
+import MarkerClusterGroup from 'react-leaflet-markercluster'
+import 'react-leaflet-markercluster/dist/styles.min.css'
+
+// Utils
+import {getColor} from './dimensionCalculations'
+import {SVGIcon} from './SVGIcon'
+import {defineToolTipEffect, formatValue} from './processing/toolTips'
+import {GeoTooltip} from './GeoTooltip'
+
+// Types
+import {GeoTable} from './processing/GeoTable'
+import {ClusterAggregation, GeoPointMapViewLayer} from '../../types/geo'
+import {Config} from '../../types'
+
+// Constants
+const MARKER_ICON_SIZE = [26, 38]
+const CLUSTER_ICON_SIZE = 30
+const CLUSTER_ICON_SIZE_BORDER = 4
+const DEFAULT_CLUSTER_COLOR = '#00a3ff'
+
+interface Props {
+  table: GeoTable
+  colorFieldName: string
+  properties: GeoPointMapViewLayer
+  stylingConfig: Partial<Config>
+  isClustered: boolean
+}
+
+const formatRowInfo = (
+  properties: GeoPointMapViewLayer,
+  table: GeoTable,
+  index
+) => {
+  const result = []
+  const timeValue = table.getTimeString(index)
+  if (timeValue)
+    result.push({
+      key: '_time',
+      name: 'Time',
+      type: 'string',
+      values: [timeValue],
+    })
+  const colorValue = table.getValue(index, properties.colorField)
+  const {colorDimension} = properties
+  const colorInfo = formatValue(
+    properties.colorField,
+    'Color',
+    colorValue,
+    colorDimension
+  )
+  if (colorInfo) result.push(colorInfo)
```

Comment:
Please always use braces for if statements. We will be updating the eslint rules to reflect this.